README.md

```diff
@@ -57,6 +57,10 @@ JSON file. You may also set the GOOGLE_APPLICATION_CREDENTIALS environment varia
     ./cloud_sql_proxy -dir=/cloudsql -instances=my-project:us-central1:sql-inst=tcp:3306 &
     mysql -u root -h 127.0.0.1
 
+    # For specifying another host:
+    ./cloud_sql_proxy -dir=/cloudsql -instances=my-project:us-central1:sql-inst=tcp:0.0.0.0:3306 &
+    mysql -u root -h 0.0.0.0
```

Comment:
I don't think specifying `-h 0.0.0.0` is going to work. Maybe you meant 127.0.0.1 still (or the actual IP of the machine running the proxy?)